saleor/product/utils.py

```diff
@@ -171,3 +176,11 @@ def price_range_as_dict(price_range):
         return None
     return {'maxPrice': price_as_dict(price_range.max_price),
             'minPrice': price_as_dict(price_range.min_price)}
+
+
+def get_variant_url_from_product(product, attributes):
```

Comment:
I need function to generate url to not existing variants which could be selected using variant picker (we can add such variants added to wishlist).